pulp_ansible/app/models.py

```diff
@@ -409,6 +409,8 @@ def issuer(self):
     @property
     def trust_updater(self):
         """Get a custom TrustUpdater instance depending on the TUF metadata repository provided."""
+        if self.tuf_url.endswith("/"):
+            self.tuf_url = self.tuf_url[-1]
```

Comment:
```suggestion
        if self.tuf_url.endswith("/"):
            self.tuf_url = self.tuf_url[:-1]
```
You could also remove the 'if' statement with `self.tuf_url = self.tuf_url.rstrip("/")`